CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-0.0.1 (TBD 2017)
-----------------
+0.1.1 (2017-11-21)
+------------------
 
-- Initial Release
+Features:
+
+- Make the entrez lookup configurable (#6)
+
+0.1.0 (2017-11-20)
+------------------
+
+Features:
+
+- Cleanup library and add features ()#3)
```

Comment:
typo: `()#3)`

Reply:
Also the URLs aren't links in the rendered markdown, might as well paste the full URL to the PRs instead?